components/pagination/spec/Pagination.stories.mdx

```diff
@@ -0,0 +1,136 @@
+import { Meta, Preview, Props, Story } from '@not-govuk/docs-components';
+import { Pagination } from '../src/Pagination';
+import readMe from '../README.md';
+
+<Meta
+  title="Pagination"
+  component={ Pagination }
+  parameters={ {
+    chromatic: { viewports: [640, 480] },
+    description: 'Component to allow separating long tables into separate tables',
+    jest: ['Pagination'],
+    notes: readMe
+  } }
+/>
+
+# Pagination
+
+A component for navigating a long list of results.
+
+<Preview>
+  <Story name="Pagination">
+    <Pagination
+      results="249"
+      resultsPerPage="25"
+      page="3"
+    />
+  </Story>
+</Preview>
+
+<Props of={ Pagination } />
+
+
+## Stories
+### Standard
+
+A standard pagination.
+
+<Preview>
+  <Story name="Standard">
+    <Pagination
+      results="249"
+      resultsPerPage="25"
+      page="3"
+    />
+  </Story>
+</Preview>
+
+
+### Start
+
+A pagination component on the first page of results.
+
+<Preview>
+  <Story name="Start">
+    <Pagination
+      results="737"
+      resultsPerPage="25"
+      page="1"
+    />
+  </Story>
+</Preview>
+
+
+### Middle
+
+A pagination component in the middle of the list.
+
+<Preview>
+  <Story name="Middle">
+    <Pagination
+      results="737"
+      resultsPerPage="25"
+      page="15"
+    />
+  </Story>
+</Preview>
+
+
+### End
+
+A pagination component on the last page of results.
+
+<Preview>
+  <Story name="End">
+    <Pagination
+      results="737"
+      resultsPerPage="25"
+      page="30"
+    />
+  </Story>
+</Preview>
+
+
+### Near end
+
+A pagination component on the second last page of results.
+
+<Preview>
+  <Story name="NrEnd">
+    <Pagination
+      results="737"
+      resultsPerPage="25"
+      page="29"
+    />
+  </Story>
+</Preview>
+
+
+### Results Per Page
+
+A pagination component with a different number of results per page.
+
+<Preview>
+  <Story name="Results Per Page">
+    <Pagination
+      results="249"
+      resultsPerPage="10"
+      page="2"
+    />
+  </Story>
+</Preview>
+
+
+### Invalid props
```

Comment:
My expectation of storybook tests is to show how a component _should_ be used, and to [visually] check that style regressions aren't introduced. I think this invalid props test should only happen at the unit level (if at all).

Reply:
Have removed both the unit tests and story for invalid props

Reply:
This now looks good to me.